css/accordion.css

```diff
@@ -18,9 +18,9 @@
 }
 
 [data-accordion-flip] {
-    transition: var(--transition-default);
+  transition: var(--transition-default);
 }
 
 .accordion__item.is-active [data-accordion-flip] {
-    rotate: -180deg;
+  rotate: -180deg;
```

Comment:
While this indentation doesn't match the rest of the file, it _does_ match the `.editorconfig` rules so I won't ask you to change it. At some stage if we really wanna be tidy we can go through and fix the rest of the indentation to be 2 spaced like this is.